### PR TITLE
removed shade from utopiColor, removed all instances of .shade() and moved actual colors to the light and dark themes

### PR DESCRIPTION
**Pull request by thetylerwolf**
**Problem:**
UtopiColor.shade() is no longer desirable functionality.

**Fix:**
Removed the .shade() function from UtopiColor and moved all colors created with it that exist outside of `theme.ts` into the `theme.ts`

**Commit Details:**
- Removed `shade` function from `utopi-color-helpers.ts`
- Removed tests using the `shade` function

### CODE REVIEW

File: editor/src/components/titlebar/title-bar.tsx [hunk 1]
```diff
@@ -28,8 +26,8 @@ const ProjectTitle: React.FC<React.PropsWithChildren<ProjectTitleProps>> = ({ ch
   return <div style={{ fontWeight: 400, fontSize: 12, padding: '0 10px' }}>{children}</div>
 }
 
-const TitleBar: React.FC<React.PropsWithChildren<TitleBarProps>> = () => {
-  const { loginState, projectName, isCodeEditorVisible, isPreviewPaneVisible, dispatch } =
+const TitleBar = React.memo(() => {
```
Comment (balazsbajorics) [inline editor/src/components/titlebar/title-bar.tsx:29]: this seems to be an unrelated fix that sneaked in :)

**Review by enidemi — APPROVED** [review]

**Review by seanparsons — APPROVED** [review]

File: editor/src/components/titlebar/title-bar.tsx (hunk 1 above)
Comment (thetylerwolf) [reply editor/src/components/titlebar/title-bar.tsx:29]: eh yeah I caught it after I pushed 😅
